packages/@jsii/go-runtime/jsii-calc-test/main.go

```diff
@@ -1,21 +1,17 @@
 package main
 
 import (
-	// "log"
 	"fmt"
-	jsiicalc "github.com/aws-cdk/jsii/jsii-calc/golang/jsiicalc"
-	"github.com/aws-cdk/jsii/jsii-experimental"
+	calc "github.com/aws-cdk/jsii/jsii-calc/golang/jsiicalc"
+	baseofbase "github.com/aws-cdk/jsii/jsii-calc/golang/scopejsiicalcbaseofbase"
 )
 
 func main() {
 	fmt.Println("Hello, JSII")
-	// client, err := jsii.InitClient()
+	fmt.Printf("JSII_AGENT=%s\n", calc.NewJsiiAgent().GetValue())
```

Comment:
Note: `GetValue` is supposed to be `static`, which I suspect is why this does not currently work.